cli: invalid error/fallback on run command input

`run` subcommand does not correct handle input when directory is present. For example the input `test` results in a Node `MODULE_NOT_FOUND` error, instead of the CLI "sub command" not found default.

```shell
$ /Users/danielhansen/project/node_modules/.bin/compas run test
node:internal/modules/cjs/loader:1056
  throw err;
  ^

Error: Cannot find module '/Users/danielhansen/project/test'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1053:15)
    at Module._load (node:internal/modules/cjs/loader:898:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:84:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

Expected:
```
$ /Users/danielhansen/project/node_modules/.bin/compas run test
14:24:15.447 error[compas] Invalid sub command 'test' for 'compas run'.
Can run files from the following places:
- Files located in the scripts directory.
- Scripts defined in the package.json
- Any path to a JavaScript file

Scripts directory:
  - api
  - queue

Package.json scripts:
  - start

error Command failed with exit code 1.
```

--- a/docs/references/cli.md
+++ b/docs/references/cli.md
@@ -37,6 +37,7 @@ is optimally configured.
 
 - Checks if the '.env.local' is in the .gitignore if it exists
 - Checks if all Compas packages are the same version
+- Checks if graphviz is available
 
 | Option     | Description                                              |
 | ---------- | -------------------------------------------------------- |
@@ -233,11 +234,11 @@ with caution.
 Run arbitrary JavaScript files, scripts defined in the package.json and scripts
 located in the scripts directory.
 
-| Option        | Description                                              |
-| ------------- | -------------------------------------------------------- |
-| --script-args | undefined (string)                                       |
-| --node-args   | undefined (string)                                       |
-| -h, --help    | Display information about the current command. (boolean) |
+| Option        | Description                                                                       |
+| ------------- | --------------------------------------------------------------------------------- |
+| --script-args | Arguments passed as is to the script when executed (like '--port 3000'). (string) |
+| --node-args   | Arguments passed to Node when executing the script (like '--inspect'). (string)   |
+| -h, --help    | Display information about the current command. (boolean)                          |
 
 ### `compas run $script`
 
@@ -246,8 +247,8 @@ The file or script to run.
 | Option        | Description                                                                                                      |
 | ------------- | ---------------------------------------------------------------------------------------------------------------- |
 | --watch       | Run the command, restarting it when file changes happen. See 'compas help watch' for more information. (boolean) |
-| --script-args | undefined (string)                                                                                               |
-| --node-args   | undefined (string)                                                                                               |
+| --script-args | Arguments passed as is to the script when executed (like '--port 3000'). (string)                                |
+| --node-args   | Arguments passed to Node when executing the script (like '--inspect'). (string)                                  |
 | -h, --help    | Display information about the current command. (boolean)                                                         |
 
 ## `compas test`

--- a/packages/cli/src/compas/commands/run.js
+++ b/packages/cli/src/compas/commands/run.js
@@ -1,4 +1,4 @@
-import { existsSync } from "fs";
+import { existsSync, statSync } from "fs";
 import path from "path";
 import { isNil, spawn } from "@compas/stdlib";
 import { collectScripts } from "../../utils.js";
@@ -23,7 +23,9 @@ export const cliDefinition = {
       dynamicValue: {
         validator: (value) => {
           const scriptCollection = collectScripts();
-          const isValid = !isNil(scriptCollection[value]) || existsSync(value);
+          const isValid =
+            !isNil(scriptCollection[value]) ||
+            (existsSync(value) && statSync(value).isFile());
 
           if (isValid) {
             return {
@@ -35,9 +37,9 @@ export const cliDefinition = {
             isValid,
             error: {
               message: `Can run files from the following places:
-- Files located in the scripts directory.
+- Files located in the 'scripts' directory.
 - Scripts defined in the package.json
-- Any path to a JavaScript file
+- Any path to a file, that can be executed via 'node [$--node-args] $file'
 
 Scripts directory:
 ${Object.entries(scriptCollection)
@@ -94,6 +96,8 @@ ${Object.entries(scriptCollection)
     {
       name: "scriptArguments",
       rawName: "--script-args",
+      description:
+        "Arguments passed as is to the script when executed (like '--port 3000').",
       value: {
         specification: "string",
       },
@@ -101,6 +105,8 @@ ${Object.entries(scriptCollection)
     {
       name: "nodeArguments",
       rawName: "--node-args",
+      description:
+        "Arguments passed to Node when executing the script (like '--inspect').",
       value: {
         specification: "string",
       },

